src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAWindowsDesktopProject.cs

```diff
@@ -35,6 +35,25 @@ public void It_fails_if_windows_target_platform_version_is_invalid()
                 .HaveStdOutContaining("NETSDK1137");
         }
 
+        [WindowsOnlyFact]
+        public void It_succeeds_if_windows_target_platform_version_has_trailing_zeros()
+        {
+            var testProject = new TestProject()
+            {
+                Name = "ValidWindowsVersion",
+                IsSdkProject = true,
+                TargetFrameworks = "net5.0"
+            };
+            testProject.AdditionalProperties["TargetPlatformIdentifier"] = "windows"; 
```

Comment:
```suggestion
            testProject.AdditionalProperties["TargetPlatformIdentifier"] = "Windows"; 
```

I thought TPI/TFI should be in PascalCasing, no?

Reply:
We do normalize the casing to `Windows`.  That said, if a test isn't meant to be testing case normalization, it should probably use `Windows` if the platform is specified via the `TargetPlatformIdentifier` property, and `-windows` if it's specified in the `TargetFramework`, as those are the standard casings in each case.